core/src/toga/style/pack.py

```diff
@@ -138,22 +143,38 @@ def reapply(self, *args, **kwargs):
         "padding_left": "margin_left",
     }
 
-    @classmethod
-    def _update_property_name(cls, name):
-        if new_name := cls.DEPRECATED_PROPERTIES.get(name):
-            cls._warn_deprecated(name, new_name, stacklevel=4)
+    _ALIASES = {
+        "horizontal_align_content": {ROW: "justify_content"},
+        "horizontal_align_items": {COLUMN: "align_items"},
+        "vertical_align_content": {COLUMN: "justify_content"},
+        "vertical_align_items": {ROW: "align_items"},
+    }
```

Comment:
Is this list of aliases correct (or, at least, consistent with the proposal in #3113)? Shouldn't there be a ROW and COLUMN interpretation of all four aliases?

Reply:
Scratch that - I'm getting myself tied in knots expecting `justify_items` and `align_content`. However, I think this plays into my bigger point on #3113.